### ISSUE TITLE
Evaluation Reports that are downloaded and have no errors are missing the Evaluation Results data

### ISSUE THREAD

**Issue by esaber76** [US-EPA-CAMD/PI-1#6429]
If I download an Evaluation Report that has no errors, the Evaluation Results section is missing from the report.

![image](https://github.com/user-attachments/assets/803b1a85-231a-4f40-8c80-f3c4889e1148)

The reports display correctly when selecting the links from the Evaluate screen, and when received as email attachments.

![image](https://github.com/user-attachments/assets/0a28f472-5788-483b-9a64-ab4308151edc)

#5959 added the appropriate message. Unsure if this update was also included for the download functionality at the time.

**Comment by ntknguyen** [comment]
@esaber76  - I tested on two configurations, Limestone 298 unit LIM1 and Cyrstal River 628 unit 1GTA, and received two results.  Limestone 298 unit LIM1 shows Critical errors in the status code but Informational in the Download report.  Crystal River 628 unit 1GTA shows Critical errors in the status code and "Evaluation has passed without error" in the Download report.

![Image](https://github.com/user-attachments/assets/7ccd1258-7174-4468-b451-40b55311c273)![Image](https://github.com/user-attachments/assets/3a755574-e7fe-4b66-a7be-706a90bdb746)


![Image](https://github.com/user-attachments/assets/01e57ac7-7f1a-4467-b3f7-f8508ab9bf04)

![Image](https://github.com/user-attachments/assets/37f6857c-fb08-4b4d-a9d7-5b13b26bf721)

**Comment by esaber76** [comment]
@ntknguyen - does clicking the critical errors link produce an evaluation report that matches the download report? I suspect the critical errors showing in the Eval Status column is a widespread problem likely related to some other change.

**Comment by ntknguyen** [comment]
> [@ntknguyen](https://github.com/ntknguyen) - does clicking the critical errors link produce an evaluation report that matches the download report? I suspect the critical errors showing in the Eval Status column is a widespread problem likely related to some other change.

Yes, the evaluation report does match the download report.  However, the notification email does not. 
![Image](https://github.com/user-attachments/assets/9afd5864-49dc-4305-b2aa-537a3b7891cd)

**Comment by esaber76** [comment]
@ntknguyen - I posted a note in the ECMPS 2.0 Teams channel. Let's see where that goes and hold off on testing of this for now. I can't imagine the issues you are seeing are related to this ticket.

**Comment by bryanramirez1** [comment]
Retested with several different configurations and confirmed that the evaluation report details appear in both the browser window as well as the downloaded report.

**Comment by bryanramirez1** [comment]
Retested with Limestone (298) and Fair Station (1218) and verified that the eval results sections shows in browser and in the downloaded report and the messages match both versions for both plants.
